source/developers/how-tos/get-your-project-docs-on-rtd.rst

```diff
@@ -0,0 +1,71 @@
+How-to get your repository docs on docs.openedx.org
+###################################################
+
+.. Titles should be imperative. https://www.grammarly.com/blog/imperative-sentences/
+
+.. How-tos should have a short introduction sentence that captures the user's goal and introduces the steps.
+
+This how-to will help you publish the documentation for your Open edX Library or Service to ReadTheDocs.org under the https://docs.openedx.org domain.
+
+Assumptions
+***********
+
+.. This section should contain a bulleted list of assumptions you have of the
+   person who is following the How-to.  The assumptions may link to other
+   how-tos if possible.
+
+* You know how to build your repo's documentation locally.
```

Comment:
Is there another doc we could link to here?

Reply:
Will add.